modules/ROOT/pages/jmx-service.adoc

```diff
@@ -125,7 +125,7 @@ HTTP 200
 
 === Adding a New JMX Publisher
 
-To add a new JMX publisher, you need to implement the Runtime Manager agent's `InternalMessageHandler` interface.
+To add a new JMX publisher, you need to implement the Runtime Manager agent `InternalMessageHandler` interface.
```

Comment:
Why do we say "you need to" here? isn't that implied in all our instructions?

Reply:
I just didn't remove it! It be gone.